config.json

```diff
@@ -328,7 +328,11 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "recursion",
+        "arrays",
```

Comment:
Could you please remove `arrays` and leave only `lists`.
In my mind `arrays` can store data only of the same type (in Java for example), but `lists` can store data of any type (like in Python or any implementation of `List` interface in Java)

Reply:
Corrected!